sample_config/sample.routing.config

```diff
@@ -1,8 +1,6 @@
 {
```

Comment:
I suspect we can get rid of this file and config handler altogether. If we wish to set Elder_size etc. we cah have `cfg` flags for tests or even add that to the builder? I think though cfg flags are easy.  Then we don't have routing config files at all.